### PR TITLE
feat: [modal migration] shared bookmarks

### PR DESCRIPTION
**Pull request by el**
## Changes

Shared bookmarks

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/BookmarkFeedLayout.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default function BookmarkFeedLayout({
 }: BookmarkFeedLayoutProps): ReactElement {
   const { user, tokenRefreshed } = useContext(AuthContext);
   const [showEmptyScreen, setShowEmptyScreen] = useState(false);
-  const [showSharedBookmarks, setShowSharedBookmarks] = useState(false);
+  const [showSharedBookmarks, setShowSharedBookmarks] = useState(true);
```
Comment (rebelchris) [inline packages/shared/src/components/BookmarkFeedLayout.tsx:39]: Think this might have been testing?

**Review by rebelchris — CHANGES_REQUESTED** [review]
One small change and we good to go 😀

File: packages/shared/src/components/BookmarkFeedLayout.tsx (hunk 1 above)
Comment (el) [reply packages/shared/src/components/BookmarkFeedLayout.tsx:39]: ahh yeap, forgot to revert

**Review by rebelchris — APPROVED** [review]
Thank you 🥳

**Pull request merged**